vault/raft.go

```diff
@@ -109,6 +110,15 @@ func (c *Core) startRaftStorage(ctx context.Context) (retErr error) {
 	var raftTLS *raft.TLSKeyring
 	switch raftTLSEntry {
 	case nil:
+		// If this is HA-only and no TLS keyring is found, that means the
+		// cluster has not been bootstrapped or joined. We return early here in
+		// this case. If we return here, the raft object has not been instantiated,
+		// and a bootstrap call should be made.
+		if c.isRaftHAOnly() {
+			c.logger.Trace("skipping raft backend setup during unseal, bootstrap must be started")
```

Comment:
```suggestion
			c.logger.Trace("skipping raft backend setup during unseal, bootstrapping must be started")
```

Reply:
Reworded this, albeit a bit differently.